ITERATION_PLAN.md

```
@@ -0,0 +1,1275 @@
+instead of HTML. Users get modern, beautiful API docs powered by VitePress
+while keeping full customization of the VitePress project.
+
+Based on dart-lang/dartdoc v9.0.2 (commit af008503).
+
+---
+
+## Architecture Overview
+
+dartdoc pipeline:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)